src/OrchardCore.Modules/OrchardCore.Markdown/Views/MarkdownField.Edit.cshtml

```diff
@@ -8,7 +8,7 @@
     var culture = await Orchard.GetContentCultureAsync(Model.Field.ContentItem);
 }
 
-<div class="form-group">
+<div class="form-group simplemde-editor">
```

Comment:
I think you have to do the same for the MarkdownBodyPart

Reply:
ok, done.